### ISSUE TITLE
Accessibility & UX review

### ISSUE THREAD

**Issue by 12** [BBC-archive/psammead#562]
**Is your feature request related to a problem? Please describe.**
We need to create the Most Read component in an AMP-first approach. This will need to follow the UX specification, and a swarm should be carried out to ensure it meets accessibility criteria. The developer(s) that picks this up should pair closely with UX.

This should work both left-to-right and right-to-left.

**Describe the solution you'd like**
- This ticket is to ensure the Most Read component within Psammead is

- accessible
- AMP-compliant
- and meets UX specification.

**Testing notes**
[Tester to complete]

Dev insight: Work with Tester, UX designer and other developers to test componet.

**Comment by pjlee11** [comment]
Along with the UX designs we should evaluate what font variations are required for the component. We should also investigate using eastern arabic numerals for the Persian if possible. It may be that the eastern arabic numerals should be handled in a follow up issue, not to bloat this piece of work.

**Comment by andrew-nowak** [comment]
# Screen reader ux

![image](https://user-images.githubusercontent.com/10963046/58816242-048e7b00-8621-11e9-8fdd-988aa09e6e28.png)

![image](https://user-images.githubusercontent.com/10963046/58816276-14a65a80-8621-11e9-9dd9-4160eef6e94e.png)

![image](https://user-images.githubusercontent.com/10963046/58816327-31db2900-8621-11e9-8bdf-02e13b39766f.png)

**Comment by pjlee11** [comment]
Just to clarify the tab order isn't the numbers above, the numbers are purely for labelling the diagram

**Comment by tanveenad** [comment]
Accessibility Acceptance criteria -

**Most Read - Landmark**
Given I use a screen reader
When I navigate to the Most Read section on an article page
Then a region landmark must be announced e.g. Region
And this must be announced with a label
And the label must be the same as the h2 (e.g. Most read) and be unique to the page
And this should not be inside the main landmark**

**Most Read - h2**
Given I use a screen reader
When I navigate to the Most Read section
Then a heading level 2 must be announced as the first piece of content
And the h2’s announced text should be the same as the region landmark’s label for this section
And the h2 must be unique to the page

**Most Read - Ordered list**
Given I use a screen reader, including VoiceOver  (Note we are particularly checking VoiceOver as there is a bug with this screen reader)
And I am viewing Most Read
When navigating to the ordered list
Then it should be announced as a list

**Most Read - List item**
Given I use a screen reader
And I am viewing Most Read
When navigating to a list item in the ordered list
Then the text should be announced
And it should be prefixed by the announcement of the list item number
Given I am viewing Most Read
When the list items are displayed in 2 columns or more
Then each column must read from left to right
Given I am viewing Most Read
When the list items are in columns
Then they must be within one ordered list
The list item number must meet the colour contrast ratio of 4.1:5

**Most Read - List item links**
Given I use a keyboard only
When I navigate to Most Read by pressing the TAB key
Then the link for each list item must be focusable
And these must have visible state change on focus
And this must not be identified by colour alone
And be in addition to the browser default focus outline styling
Given I use a mouse
When I hover over the list item numberThen the mouse cursor will change from a pointer to a hand
And the link must have a visible state change
And this must not be identified by colour alone

**Comment by jamesbhobbs** [comment]
See notes here (on what could be reusable) https://github.com/bbc/psammead/issues/569

**Comment by DenisHdz** [comment]
This can be closed as done in https://github.com/bbc/simorgh/issues/6185
